CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
 * Conform `@ThreadSafe` and `ThreadSafeReference` to `Sendable`.
+* Add ability to use Swift Query syntax in `@ObservedResults`.
```

Comment:
Will be nice to have an example here something like
* Add ability to use Swift Query syntax in @ObservedResults, which allows to filter results collection using `where`. 
and maybe adding an example.
I'm not very good with words so feel free to make it better.